### DISCUSSION TITLE
IS_BROWSER in example

### DISCUSSION THREAD

**Discussion by o-az** [freshframework/fresh#279] [General]
What's the point of disabling [this button in the example](https://github.com/lucacasonato/fresh/blob/97238ef5ac7d5cfe660c97794f6f34151b873754/examples/counter/islands/Counter.tsx#L15) when `IS_BROWSER` is !true?

Currently my buttons are not working. I have them in a components folder and import them to pages/other components. I'm trying understand why and think this relates somehow.

**Comment by lucacasonato** [comment]
Because until the client side code for the component is loaded (until `IS_BROWSER` is true), the button will not do anything.

As to why it is not working: is your component that you want client side interaction for an [island](https://fresh.deno.dev/docs/getting-started/adding-interactivity)?

**Reply by o-az** [reply]
Yup it was. Is there a way to use an Island as a wrapper component that I can use to make any component interactive? Sounds hacky but do you think it will work? Tried testing it but no luck.
